packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerForm/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import { Image } from 'react-native';
+import { Formik } from 'formik';
+import {
+  ModalView,
+  Switch,
+  TextInput,
+  Button,
+  styled,
+  FlexedView,
+  PaddedView,
+} from '@apollosproject/ui-kit';
+import AddPrayerHeader from '../AddPrayerHeader';
+
+const StyledButton = styled(() => ({
+  borderRadius: 0,
+}))(Button);
+
+const BottomView = styled(() => ({
+  justifyContent: 'flex-end',
+}))(FlexedView);
+
+const SwitchContainer = styled(() => ({
+  width: '60%',
+  alignSelf: 'center',
+}))(PaddedView);
+
+const AddPrayerForm = memo(({ imgSrc, btnLabel, ...otherProps }) => (
+  <Formik
+    initialValues={{ prayer: '', anonymous: false }}
+    onSubmit={(values) => console.log(values)}
+  >
+    {({ handleChange, handleBlur, handleSubmit, values }) => (
+      <ModalView {...otherProps}>
+        <FlexedView>
+          <AddPrayerHeader imgSrc={imgSrc} />
+          {/* TODO: add underline={false} prop to TextInput, 
+        pending https://github.com/ApollosProject/apollos-prototype/issues/629 */}
+          <PaddedView>
+            <TextInput
+              editable
+              placeholder="Start typing your prayer..."
+              onChangeText={handleChange('prayer')}
+              onBlur={handleBlur('prayer')}
+              value={values.prayer}
+            />
+          </PaddedView>
+          <BottomView>
+            <SwitchContainer>
+              <Switch
+                label="Share anonymously"
```

Comment:
This label isn't our primary color.

Reply:
does it need to be?

Reply:
Is there a `color` prop on the `Switch`? If not, we'd have to bring over the whole component and edit it just to change the color. Maybe a PR to core to allow a color prop so we could use whatever color we wanted for the label in the future would be the right call here?